### DISCUSSION TITLE
Extensions don't show Github Copilot on Visual Studio 2022.

### DISCUSSION THREAD

**Discussion by Themprogrammer** [community/community#14678] [Copilot Conversations]
I've attempted to look up "Github Copilot" on the Extensions page on Visual Studio 2022, and it doesn't show anything.

**Comment by kzu** [comment]
Make sure you have selected the Online tab before performing the search. Also, you need to be on the latest released stable version of Visual Studio 2022 (17.1 or later) in order to see it:

![image](https://user-images.githubusercontent.com/169707/162799091-b5fcb421-1dd0-461f-83d1-54ce9ac131f3.png)

**Reply by Themprogrammer** [reply]
thank you , i will try that

**Reply by Themprogrammer** [reply]
o it works now thx

**Reply by RecluseIndustries** [reply]
Not working for me, using VS 2022,  version 17.3.4
![Screenshot (4)](https://user-images.githubusercontent.com/6449667/222108596-15291aae-717e-43a8-9cc6-5c70d7d0e0de.png)

**Reply by neckell** [reply]
Same here
![image](https://user-images.githubusercontent.com/13804182/225051628-27fe6a2b-6518-4d0b-8a58-27478431189e.png)

**Comment by NickStrupat** [comment]
@Themprogrammer Install it manually if you still can't see it.

https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs

I don't see it in the search results on VS 2022 17.4.0 Preview 2.1, but I installed manually and it seems to work.

**Reply by RecluseIndustries** [reply]
Requires version 17.4.4 to show up in search or to install

**Comment by ttandel1** [comment]
I still don't see Github Copilot in Extension of VS2022 17.3.5

**Reply by skay97** [reply]
After experiencing the same issue, I've realized that the extension simply isn't compatible with arm architecture(I am using an arm installation of windows on my M2 mac). If you're using something similar, this could be why.
https://github.com/community/community/discussions/43779

**Reply by skay97** [reply]
Since the extension still hasn't been officially ported by Microsoft, you can do it manually like so: 
https://stackoverflow.com/a/76737023
Copilot now works on my arm installation of visual studio in parallels.
